data-science-onramp/ai-platform/requirements.txt

```diff
@@ -0,0 +1,5 @@
+# Version list reflects AI platform runtime 2.2
+# More information here https://cloud.google.com/ai-platform/training/docs/runtime-version-list
+google-api-python-client==1.12.1
+google-cloud-storage==1.29.0
+oauth2client<4
```

Comment:
in the runtime version list it looks like they use oauth2client 4.1.3? Should we be doing that? If we can pin, that would be preferred, because renovate bot really hates unpinned dependencies